docs/examples/forecasting.py

```diff
@@ -0,0 +1,399 @@
+"""
+Forecasting
+============
+
+On this example we present a full forecasting workflow for the Sioux Falls
+example model.
+"""
+
+## Imports
+from uuid import uuid4
+from tempfile import gettempdir
+from os.path import join
+from aequilibrae.utils.create_example import create_example
+from aequilibrae import logger
+import logging
+import sys
+
+# %%
+# We create the example project inside our temp folder
+fldr = join(gettempdir(), uuid4().hex)
+
+project = create_example(fldr)
+
+# We the project open, we can tell the logger to direct all messages to the terminal as well
+stdout_handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter("%(asctime)s;%(name)s;%(levelname)s ; %(message)s")
+stdout_handler.setFormatter(formatter)
+logger.addHandler(stdout_handler)
+
+# %% md
+
+## Traffic assignment with skimming
+
+# %%
+
+from aequilibrae.matrix import AequilibraeMatrix
+from aequilibrae.paths import TrafficAssignment, TrafficClass
+
+# %%
+
+# we build all graphs
+project.network.build_graphs()
+# We get warnings that several fields in the project are filled with NaNs.  Which is true, but we won't use those fields
+
+# %%
+
+# we grab the graph for cars
+graph = project.network.graphs['c']
+
+# let's say we want to minimize free_flow_time
```

Comment:
... minimise time, the uncongested value is given in free_flow_time. Splitting hairs :)

Reply:
American English here, please!  ;-)  But I agree with you!